sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/tasks/TaskManagementSteps.java

```diff
@@ -96,6 +99,42 @@ public TaskManagementSteps(
           webDriverHelpers.clickOnWebElementBySelector(getByEventUuid(eventUuid));
         });
 
+    When(
+        "^I filter Task context by ([^\"]*)$",
+        (String filterType) -> {
+          webDriverHelpers.selectFromCombobox(TASK_CONTEXT_COMBOBOX, filterType);
+          webDriverHelpers.clickOnWebElementBySelector(APPLY_FILTER);
+          webDriverHelpers.waitUntilIdentifiedElementIsPresent(TABLE_ROW);
+        });
+
+    When(
+        "^I am checking if filter works correctly for ([^\"]*)$",
+        (String filterType) -> {
+          webDriverHelpers.checkWebElementContainsText(FILTER_ACTIVE_TASK_CONTEXT, filterType);
```

Comment:
let's check all entries displayed in table, because I remember in the past that maybe the first 2-3 results were ok, then the rest were different, so the check was invalid
Also, the name, let's say: I check displayed task's context is ([^\"]*

Reply:
when we have a lot of data, checking will be take a lot of time, maybe we should check first 10 ?